src/BuildingRegistry.Api.Legacy/appsettings.json

```diff
@@ -11,6 +11,12 @@
     "ServiceName": "BuildingRegistry - dev"
   },
 
+  "Grb": {
+    "Wfs": {
+      "Url": "https://geoservices.informatievlaanderen.be/overdrachtdiensten/GRB/wfs"
```

Comment:
Stupid question, but does staging and production go to the same URL? Or do they also have a beta WFS?

Reply:
It does for now. This was hard coded before, so no change in behaviour. 
But if we need to change it, it's now configurable.